rust/Makefile

```diff
@@ -59,11 +60,56 @@ alloc-cfgs = \
     --cfg no_rc \
     --cfg no_sync
 
+proc_macro2-skip_flags := \
```

Comment:
While working on kproc-macros https://github.com/vincenzopalazzo/kproc-macros and see that prob_macros2 can be optional in projects like Linux Kernel (see https://github.com/dtolnay/syn/blob/master/Cargo.toml#L39)

I think we do not need to include `proc_macro2` because it is used just for projects that import syn and quote with cargo.

P.: Well I should not say that if I want kproc-macros in right? 😸

Reply:
https://github.com/dtolnay/syn/blob/master/Cargo.toml#L39 doesn't say that proc-macro2 is optional. It says that the default features are disabled for it.

Reply:
Uh! you are right! damnt! I think I take a look at the wrong Cargo, maybe it was the https://github.com/vincenzopalazzo/kproc-macros/blob/main/kproc-parser/Cargo.toml#L13 

Apologies!